app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -123,7 +120,12 @@ public class LoyaltyCardViewActivity extends CatimaAppCompatActivity implements
     FloatingActionButton editButton;
 
     Guideline centerGuideline;
-    SeekBar barcodeScaler;
+    SeekBar barcodeHeightScaler;
+    SeekBar barcodeWidthScaler;
+    TextView zoomHeightText;
+    TextView zoomWidthText;
+    LinearLayout widthScalerLayout;
+    LinearLayout heightScalerLayout;
```

Comment:
Please be consistent in order.

Now it is:
Height
Width
Height
Width
Width
Height